Bootstrap/app/src/main/res/values/dimens.xml

```diff
@@ -118,10 +118,8 @@
     <dimen name="home_menu_marginStart">72dp</dimen>
     <dimen name="home_menu_marginTop">56dp</dimen>
 
-
-
-
-
+    //progress dialog
+    <dimen name="progress_dialog_fullProgress">100</dimen>
```

Comment:
The name of the dimens makes reference to the dialog , Is not necessary a comment in this case